src/test/java/racing/domain/CarTest.java

```diff
@@ -0,0 +1,21 @@
+package racing.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import racing.domain.Car;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class CarTest {
+
+    @DisplayName("move() 호출시 거리값이 잘 계산되어 나오는지 확인")
+    @Test
+    void distanceRangeCheck() {
+        // when
+        Car car = new Car(() -> CarMover.GO_FORWARD);
```

Comment:
```suggestion
        // given
        Car car = new Car(() -> CarMover.GO_FORWARD);
        
        // when
```

given 절에 해당하는게 맞는것 같아보여